searx/templates/simple/result_templates/images.html

```diff
@@ -2,6 +2,7 @@
         <a {% if results_on_new_tab %}target="_blank" rel="noopener noreferrer"{% else %}rel="noreferrer"{% endif %} href="{{ result.img_src }}">{{- "" -}}
                 <img class="image_thumbnail" {% if results_on_new_tab %}target="_blank" rel="noopener noreferrer"{% else %}rel="noreferrer"{% endif %} src="{% if result.thumbnail_src %}{{ image_proxify(result.thumbnail_src) }}{% else %}{{ image_proxify(result.img_src) }}{% endif %}" alt="{{ result.title|striptags }}" loading="lazy" width="200" height="200">{{- "" -}}
                 <span class="title">{{ result.title|striptags }}</span>{{- "" -}}
+                <span class="title source">{{ result.parsed_url.netloc }}</span>{{- "" -}}
```

Comment:
`<span class="source">`  and then 

```less
span.source,
span.title {
  ...
}
```

so it is easy to pick either the title or the source from javascript or CSS.